.github/CODEOWNERS

```diff
@@ -5,7 +5,7 @@
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
 # The @googleapis/api-contact-center-insights is the default owner for changes in this repo
-*                       @googleapis/yoshi-java @googleapis/api-contact-center-insights
+*                       @googleapis/yoshi-java
 
 # The java-samples-reviewers team is the default owner for samples changes
-samples/**/*.java       @googleapis/java-samples-reviewers @googleapis/api-contact-center-insights
```

Comment:
Can we keep @googleapis/api-contact-center-insights as owners for samples changes? It has been the eng team's responsibility to write code samples, so keeping the eng team here gives us more flexibility in contributing samples code

Reply:
Reverted this line.